tools/news/template.rst

```diff
@@ -1,6 +1,6 @@
 {% set underline = "=" %}
-{{ top_line }}
```

Comment:
@pradyunsg @sbidoul FYI this change has completely removed the draft release title. See: https://pip.pypa.io/en/stable/news/

Reply:
Ah, that is interesting. I wanted to fix the release process which produced a duplicate title.
From a quick look at the towncrier source code I could not find a way to make it work consistently in draft and non draft mode when `title_format` is set in `[tools.towncrier]`. There must be something I misunderstand in towncrier... or it is bug, idk. Or perhaps the part of our toolchain that uses it in draft mode (where is that?) could generate the title?

Reply:
Yea, well, I prefer that to a duplicated heading in the release docs. :)

Reply:
@sbidoul this is probably a towncrier bug because the behavior changed with the latest release.

Reply:
The Towncrier regression makes normal and draft runs produce different outputs: https://github.com/twisted/towncrier/issues/105. Possible upstream fix: https://github.com/twisted/towncrier/pull/303.